.github/workflows/mac.yml

```diff
@@ -0,0 +1,23 @@
+name: mac
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: macos-10.15
+    steps:
+    - uses: actions/checkout@v2
+    - name: Download ROOT
+      run: |
+        curl -O https://root.cern/download/root_v6.20.02.macosx64-10.15-clang110.tar.gz
+        tar xf root_v6.20.02.macosx64-10.15-clang110.tar.gz
+        sudo pip install yaml
```

Comment:
```suggestion
        sudo pip install pyyaml
```

Reply:
I am checking what else is missing, this needs a requirements.txt file

Reply:
ok all imports 
```py
set(['string', 're', 'optparse', 'ROOT', 'unittest', 'subprocess', 'yaml', 'podio_config_reader', 'collections', 'EventStore', 'pickle', 'podio_templates', 'copy', 'os'])
```
only `yaml` is not in the basic python or ROOT or podio